docs/guides/multiple-documents.md

```diff
@@ -0,0 +1,107 @@
+# Serving multiple OAS documents
+
+A single Prism instance serves one OAS document.
+In order to serve multiple documents, you can use `docker-compose` utility.
+
+## Preparing configuration files
+
+Depending on your needs, you might have a `docker-compose.yaml` file like this:
+
+```yaml
+version: "3"
+services:
+  proxy:
+    image: nginx:alpine
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf
+    ports:
+      - "8080:80"
+    depends_on:
+      - prism_1
+      - prism_2
+  prism_1:
+    image: stoplight/prism:3
+    command: mock -p 4010 --host 0.0.0.0 https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v2.0/yaml/petstore.yaml
+  prism_2:
+    image: stoplight/prism:3
+    command: mock -p 4010 --host 0.0.0.0 https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore.yaml
+```
+
+And the corresponding `nginx.conf` file:
+
+```
+worker_processes auto;
+
+events {
+  worker_connections 512;
+}
+
+http {
+  server {
+    listen 80;
+
+    location /app_1 {
+      rewrite ^/app_1/(.*) /$1 break;
+      proxy_pass http://prism_1:4010/;
+    }
+
+    location /app_2 {
+      rewrite ^/app_2/(.*) /$1 break;
+      proxy_pass http://prism_2:4010/;
+    }
+  }
+}
+```
```

Comment:
We need to put a note saying this Nginx file is not complete; this is missing different required things when proxying stuff such as IP address re-write, Forward headers and stuff.

Reply:
https://github.com/stoplightio/prism/pull/730/files#diff-dcfe973ffec48a8862b3e311c8f6571bR57